web/static/js/__tests__/SidebarCategory-test.jsx

```diff
@@ -12,13 +12,13 @@ var TEST_CATEGORY = {
   "tags": [
     {
       "count": 15,
-      "id": "color-scheme",
```

Comment:
Oh, I added these by hand after copy-pasting some JSON! I guess that's not what our lint rules like. :)